### ISSUE TITLE
RCS0056 Line to long is not detected for concatenated strings

### ISSUE THREAD

**Issue by Sonic198** [dotnet/roslynator#1217]
**Product and Version Used**: 
Roslynator.Analyzers 4.5.0
Roslynator.Formatting.Analyzers 4.5.0
**Steps to Reproduce**:
Create variable
```csharp
    string test = 
        "                                                                                                                                                                                                                 "
        + "                   ";
```

**Actual Behavior**:
RCS0056 is not detected

**Expected Behavior**:
RCS0056 should be detected
